libraries/rush-lib/src/api/CommandLineConfiguration.ts

```diff
@@ -125,6 +125,13 @@ export interface IPhasedCommandConfig extends IPhasedCommandWithoutPhasesJson, I
    * If undefined, then Rush does not define a built-in "--install" CLI flag for this command and no installation is performed.
    */
   alwaysInstall: boolean | undefined;
+
+  /**
+   * If set to `true`, then this phased command will skip any checks for installation status. It will also skip loading of
+   * "rush-project.json" files. This is useful for commands that are used as custom dependency installers.
```

Comment:
What are the consequences of not loading **rush-project.json** files?  Will some APIs report errors? Will some API objects be undefined?  Or will certain settings be actually incorrect?

Reply:
Anything that depends on rush-project.json for configuration will be treated as having no configuration. Currently this means the build cache will be treated as "not configured", and any `incrementalBuildIgnoredGlobs` values will not be accounted for.

Reply:
Updated the comment here to be clear.